vulnerabilities/importers/redhat.py

```diff
@@ -17,30 +17,33 @@
 #  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
 #  VulnerableCode should be considered or used as legal advice. Consult an Attorney
 #  for any legal advice.
-#  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
+#  VulnerableCode is a free software code from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
-import requests
+
+import json
 
 from packageurl import PackageURL
+import requests
 
 from vulnerabilities.data_source import Advisory
 from vulnerabilities.data_source import DataSource
 from vulnerabilities.data_source import DataSourceConfiguration
 from vulnerabilities.data_source import Reference
+from vulnerabilities.data_source import VulnerabilitySeverity
+from vulnerabilities.severity_systems import scoring_systems
 
 
 class RedhatDataSource(DataSource):
     CONFIG_CLASS = DataSourceConfiguration
 
     def __enter__(self):
-
         self.redhat_response = fetch()
 
     def updated_advisories(self):
         processed_advisories = []
         for advisory_data in self.redhat_response:
-            processed_advisories.append(to_advisory(advisory_data))
+            processed_advisories.extend(to_advisory(advisory_data))
```

Comment:
This is likely incorrect as https://github.com/nexB/vulnerablecode/blob/783292b44181f1ad0abaf650ff1d7cac95d26c7e/vulnerabilities/importers/redhat.py#L70 return a single `Advisory` instance here:
https://github.com/nexB/vulnerablecode/blob/783292b44181f1ad0abaf650ff1d7cac95d26c7e/vulnerabilities/importers/redhat.py#L129 and not a list or iterable that could be used in `extend()`